collector/collector/config.py

```diff
@@ -25,9 +25,41 @@ class Config(object):
     SQLALCHEMY_TRACK_MODIFICATIONS = True
     LOG_FILE = 'handler.log'
 
-    # The maximum retention time for records stored in the database, measured
-    # from the time received by the `collector` app.
-    MAX_WEEK_KEEP_RECORDS = 5
+    # When PURGE_OLD_RECORDS == True then a purging system of old records will
+    # be triggered daily. If this variable is not present, then no purging will be done.
+    # If the purging system is enabled, then the following two variables must be set
+    PURGE_OLD_RECORDS = True
+    # The maximum retention time in days for records stored in the database
+    # which do not match the filters in PURGE_FILTERED_RECORDS.
+    # Use 0 to avoid deletion of all unfiltered records.
+    MAX_DAYS_KEEP_UNFILTERED_RECORDS = 30
```

Comment:
This value should match the collector default value (see my comment for shared/model.py).

Reply:
Yes